README.md

```diff
@@ -110,15 +110,15 @@ There are two VMs in the current version of Gorgonia:
 
 They function differently and take different inputs. The `TapeMachine` is useful for executing expressions that are generally static (that is to say the computation graph does not change). Due to its static nature, the `TapeMachine` is good for running expressions that are compiled-once-run-many-times (such as linear regression, SVM and the like).
 
-The `LispMachine` on the other hand was designed to take a graph as an input, and executes directly on the nodes of the graph. If the graph change, simply create a new lightweight `LispMachine` to execute it on. The `LispMachine` is suitable for tasks such as creating recurrent neural networks without a fixed size.
+The `LispMachine`, on the other hand, was designed to take a graph as an input and executes directly on the nodes of the graph. If the graph change, simply create a new lightweight `LispMachine` to execute it on. The `LispMachine` is suitable for tasks such as creating recurrent neural networks without a fixed size.
 
-Prior to release of Gorgonia, there was a third VM - a stack based VM that is similar to `TapeMachine` but deals with artificial gradients better. It may see light of day again, once this author has managed to fix all the kinks.
+Prior to the release of Gorgonia, there was a third VM - a stack-based VM that is similar to `TapeMachine` but deals with artificial gradients better. It may see the light of day again, once this author has managed to fix all the kinks.
 
 ##Differentiation##
 
 Gorgonia performs both symbolic and automatic differentiation. There are subtle differences between the two processes. The author has found that it's best to think of it this way - Automatic differentiation is differentiation that happens at runtime, concurrently with the execution of the graph, while symbolic differentiation is differentiation that happens during the compilation phase. 
 
-Runtime of course, refers to the execution of the expression graph, not the program's actual runtime.
+Runtime, of course, refers to the execution of the expression graph, not the program's actual runtime.
```

Comment:
Same thing as above re: extraneous commas. A superior version would be to remove "of course"